packages/apps/docs/src/docs/build/authentication/create-transactions.md

```diff
@@ -134,14 +134,14 @@ a Kadena SpireKey wallet might look like for the `n_eef68e581f767dd66c4d4c39ed92
 }
 ```
 
-In this example, the `n_eef68e581f767dd66c4d4c39ed922be944ede505` namespace is the namespace for the Kadena SpireKey wallet deployed on the Kadena test and main networks.
+In this example, the `n_eef68e581f767dd66c4d4c39ed922be944ede505` namespace is the namespace for Kadena SpireKey deployed on the Kadena test and main networks.
 If you deploy your own versions of the `webauthn-wallet` and `webauthn-guard` contracts, you'll use a different namespace. 
-To construct a transaction similar to this example in your application, you can use the `c:` account and `WEBAUTHN` public key returned by the Kadena SpireKey wallet in the `accountName` and `credentials` properties from the account details for a user object.
+To construct a transaction similar to this example in your application, you can use the `c:` account and `WEBAUTHN` public key returned by Kadena SpireKey in the `accountName` and `credentials` properties from the account details for a user object.
 
 ## Send data to the SpireKey wallet
 
 After you construct the unsigned transaction in your application, you can send
-that data along with a `returnUrl` to the Kadena SpireKey wallet. Kadena
+that data along with a `returnUrl` to Kadena SpireKey. Kadena
 SpireKey handles the signing process and redirects users back to your
 application. As with registration and authentication, the information is passed
 using URL parameters.
```

Comment:
This is no longer correct and should be placed as searchParams behind a `#` instead of `?`

Reply:
The same applies for the return of the tx to the dApp